LOAN/RealEstateLoans/HomeMortgageDisclosureActCoveredMortgages.rdf

```diff
@@ -24,7 +24,7 @@
 	<!ENTITY sm "http://www.omg.org/techprocess/ab/SpecificationMetadata/">
 	<!ENTITY xsd "http://www.w3.org/2001/XMLSchema#">
 ]>
-<rdf:RDF xml:base="https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanContracts/LoanHMDA/"
+<rdf:RDF xml:base="https://spec.edmcouncil.org/fibo/ontology/LOAN/RealEstateLoans/HomeMortgageDisclosureActCoveredMortgages/"
```

Comment:
Must have a better name for this, since legislation may change, and the definition says "and related"
 so there must be some rationale. Also the name is very jurisdiction-specific.

Reply:
@rivettp - the entire ontology is jurisdiction specific, and needs to be (1) moved to a jurisdiction module, and (2) reviewed with an SME.  Will reach out to a couple of banks to see if we can get SMEs to participate in a new FCT for loans